src/pages/MainRegistration.tsx

```diff
@@ -1,11 +1,18 @@
 import React, { useEffect, useState } from "react";
 import { Typography } from "@material-ui/core";
-import FamilyAPI, { FamilyListResponse } from "../api/FamilyAPI";
+import FamilyAPI from "../api/FamilyAPI";
+import { FamilyListResponse } from "../api/types";
 import FamilyTable from "../components/families/FamilyTable";
+import { DefaultField } from "../types";
 import { DefaultFields } from "../constants/DefaultFields";
 
 const MainRegistration = () => {
   const [families, setFamilies] = useState<FamilyListResponse[]>([]);
+  const FamilyEnrolmentFields: DefaultField[] = [
+    DefaultFields.IS_ENROLLED,
+    DefaultFields.CURRENT_PREFERRED_CLASS,
+    DefaultFields.CURRENT_CLASS,
+  ];
```

Comment:
let's define this outside the component, or just throw the array directly in to the props like this:
```
      <FamilyTable
        families={families}
        enrolmentFields={[DefaultFields.ENROLLED, DefaultFields.CURRENT_CLASS]}
        enrolmentFields={[
          DefaultFields.IS_ENROLLED,
          DefaultFields.CURRENT_PREFERRED_CLASS,
          DefaultFields.CURRENT_CLASS,
        ]}
        shouldDisplayDynamicFields
      />
```